src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -21,46 +36,715 @@ const ActorInit En_Ginko_Man_InitVars = {
+ActorAnimationEntry D_80A65D60[] = {
+{0x060008C0, 1.0f, 0.0f, 0.0f, 0, -4.0f,}, // smacking the floor
+{0x060043F0, 1.0f, 0.0f, 0.0f, 0, -4.0f,}, // after stamp reach, returning
+{0x06004F40, 1.0f, 0.0f, 0.0f, 2, -4.0f,}, // stamp reach
+{0x06000AC4, 1.0f, 0.0f, 0.0f, 0, -4.0f,}, // advertising / dog-paddling and looking around / "hey crowd, come check this out"
+{0x06004A7C, 1.0f, 0.0f, 0.0f, 0, -4.0f,}, // hands up staring at link `oh a small puppy` kind of pose "wow!"
+};
+
+void EnGinkoMan_Init(Actor *thisx, GlobalContext *globalCtx) {
+    EnGinkoMan* this = THIS;
+    this->actor.targetMode = 1;
+    this->actor.uncullZoneForward = 400.0f;
+    Actor_SetScale(&this->actor, 0.01f);
+    this->actor.colChkInfo.cylRadius = 100;
+    this->curTextId = 0;
+    this->newAccountFlag = 0;
+    this->stampChecked = 0;
+    this->choiceDepositWithdrawl = GINKOMAN_CHOICE_RESET;
+    this->serviceFee = 0;
+    SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_0600C240, &D_060043F0, &this->limbDrawTbl, &this->transitionDrawTbl, 0x10);
+    func_80A64554(this);
+}
+
+void EnGinkoMan_Destroy(Actor *thisx, GlobalContext *globalCtx) { }
+
+// setup wait for dialogue
+void func_80A64554(EnGinkoMan* this) {
+
+    this->actor.flags |= 1; // targetable
+    func_800BDC5C(&this->skelAnime, D_80A65D60, 1);
+    this->actionFunc = func_80A645A4;
+}
+
+//action func: waiting for dialogue
+void func_80A645A4(EnGinkoMan *this, GlobalContext *globalCtx) {
+    s32 dYaw;
+    s32 dYawABS;
+
+    dYaw = this->actor.yawTowardsPlayer - this->actor.shape.rot.y;
+    func_80A65988(this, globalCtx);
+    if (func_800B84D0(&this->actor, globalCtx)) { // Listen for dialogue start?
+        if ((gSaveContext.perm.bankRupees & 0xFFFF) == 0) {
+            func_800BDC5C(&this->skelAnime, D_80A65D60, 0);
+            func_801518B0(globalCtx, 0x44C, &this->actor);
+            this->curTextId = 0x44C; // would you like to make an account
+        } else {
+            func_800BDC5C(&this->skelAnime, D_80A65D60, 1);
+            if ((((s32) gSaveContext.perm.day % 5) == 3) && (gSaveContext.perm.isNight == 1)) {
+                func_801518B0(globalCtx, 0x467, &this->actor);
+                this->curTextId = 0x467; // "What's this? You need somethin' on a day like this?
+            } else {
+                func_801518B0(globalCtx, 0x466, &this->actor);
+                this->curTextId = 0x466; // What's this? You need somethin'?
+            }
+        }
+        func_80A65490(this); // switch dialogue actionfunc
+    } else {
+
+        dYawABS = dYaw < 0 ? -dYaw : dYaw;
+        if (dYawABS < 0x1555) {
+            func_800B8614(&this->actor, globalCtx, 100.0f);
+        }
+    }
+}
+
+#if NON_EQUIVELENT
+// non-equiv: case 14 is possibly still non-equiv but def does not match, hard to tell 
+// if else inside of jump tables makes mips2c confused, also late rodata
+// action func: non-input dialogue
+void func_80A646F4(EnGinkoMan *this, GlobalContext *globalCtx) { // 80418E34
```

Comment:
`EnGinkoMan_DepositDialogue`
This seems to be the function that handles dialogue for what the banker says after you make a deposit.